visualizer/model_dataclass.py

```diff
@@ -0,0 +1,62 @@
+from typing import List, Set, Tuple, Dict
+from dataclasses import dataclass, field
+
+import marshmallow_dataclass
+import marshmallow.validate
+import json
+
+DATA_ROOT = "/home/azerila/dev/cap/temp" # Full file path would be <DATA_ROOT>/testName/2022-03-15_19.32.28/0.json
```

Comment:
The user would have to manually change this, considering that this path would most likely not be ideal for them